docs/merged_objects.md

```diff
@@ -0,0 +1,86 @@
+# Merged objects
+
+These objects contain processed gene expression data, cell and gene metrics, associated metadata, and, in the case of multimodal data like ADTs from CITE-seq experiments, data from additional cell-based assays for all libraries in a given ScPCA project.
+See {ref}`the section on merged object processing <processing_information:merged objects` for more information on how these objects were prepared.
+
+Merged objects are provided in two formats:
+  - As an RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.17/OSCA.intro/the-singlecellexperiment-class.html) for use in R.
+  - As an HDF5 file containing an [`AnnData` object](https://anndata.readthedocs.io/en/latest/index.html) for use in Python.
+
+
+Below we present some details about the specific contents of the objects we provide.
+
+## Components of a SingleCellExperiment merged object
+
+To begin, you will need to load the `SingleCellExperiment` package and read the RDS file:
+
+```r
+library(SingleCellExperiment)
+merged_sce <- readRDS("SCPCP000000_merged.rds")
+```
+
+### SingleCellExperiment expression counts
+
+The `counts` and `logcounts` assays of the `SingleCellExperiment` object for single-cell and single-nuclei experiments contains the primary RNA-seq expression data.
+The `counts` assay contains raw counts represented as integers, and the `logcounts` assay contains normalized (on a per-library basis) counts as described in {ref}`the data post-processing section <processing_information:processed gene expression data>`.
```

Comment:
I think you need to emphasize and explicitly say that this contains combined counts and logcounts for all libraries from a given project. Other than the `per-library basis` note there's no real distinction here to tell me that what I'm looking at is merged counts.